kerl

```diff
@@ -33,18 +33,59 @@ DOCSH_GITHUB_URL='https://github.com/erszcz/docsh.git'
 ERLANG_DOWNLOAD_URL='https://erlang.org/download'
 KERL_CONFIG_STORAGE_FILENAME='.kerl_config'
 
+# Check if tput is available for colorize
+tp=$(tput -V 1>/dev/null 2>&1 && echo -n "OK" || echo -n "")
+if [ -z "$tp" ]; then
+    stderr "Colorization disabled as 'tput' (via 'ncurses') seems to be unavailable"
+    KERL_COLOR_AVAILABLE=0
+    KERL_COLORIZE=0         # Force disabling colorization
+else
+    KERL_COLOR_AVAILABLE=1
+fi
+
 # Redirect to stderr only if it is a terminal, otherwise mute
 stderr()
 {
 if [ -t 1 ] ; then
-    echo "$*" 1>&2
+    if [ -z "$l" ] || [ $KERL_COLORIZE -eq 0 ]; then
+        echo "$*" 1>&2
+    else
+        left="$(colorize $l)"
+        right="$(tput setaf ${KERL_COLOR_D:=9})"
```

Comment:
@crownedgrouse Why is KERL_COLOR_D used here? Shouldn't it rather be `tput sgr0`?

Reply:
9 is the default color of the terminal, but sgr0 could be an option. 
KERL_COLOR_D can be set to this value in your env.  Feel free to do a PR to change this.

Reply:
intention was to let extra display (not stderr) to be colored with a specific color between stderrs. using sgr0 do not allow this, but this not fundamental.

Reply:
replacing default 9 by tput sgr0 could do anyway the trick and allowing to color inbetween by setting KERL_COLOR_D anyway.

Reply:
Are you sure that 9 is the default? If I do `tput setaf 9` in my terminal it returns `\e[91m` which is a bright red color.

Reply:
Seems like what 9 means varies with what `TERM` is set to. If I do:

```erlang
Eshell V13.0  (abort with ^G)
1> os:cmd("TERM=xterm tput setaf 9").
"\e[39m"
2> os:cmd("TERM=xterm-256color tput setaf 9").
"\e[91m"
```